### ISSUE TITLE
Can't make it work on V16 - Windows 10 

### ISSUE THREAD

**Issue by Ganbin** [miyako/4d-plugin-shell-execute#1]
Hello,

This plugin seems really nice, but unfortunatly, I can't make it to work on my V16 on Windows 10.

I have downloaded your example.

**_The first issue I have is that the `Find best application` return an empty string._**

_**Second issue is that the `PRINT WITH APPLICATION` seems not to work.**_

I have this code : 
````
$docPath:=Get 4D folder(Current resources folder)+"sample.pdf"
$appPath:=Find best application ($docPath) // doesn't work
$appPath:="C:\\Program Files (x86)\\Adobe\\Acrobat Reader DC\reader\\AcroRd32.exe" // hardcoded
OPEN WITH APPLICATION ($docPath;$appPath) // work
PRINT WITH APPLICATION ($docPath;$appPath) // doesn't work
````

So the `OPEN WITH APPLICATION` is working but not the `PRINT WITH APPLICATION`

I have tried my default printer and it work.

Is there a way to configure which printer to use or it should use the default printer?

Could you help me to solve this issue?

Thanks

**Comment by miyako** [comment]
for the ``print`` command to work, the application must support the ``print`` verb. when you right-click the document, do you see a ``print`` option?

https://msdn.microsoft.com/en-us/library/windows/desktop/bb762153(v=vs.85).aspx

**Comment by Ganbin** [comment]
Ok Right, I have make it work. Here is what I have done:

Acrobat was not the default application for PDF. I have set up acrobat to be the default application for PDF and then the print option turn available on the document.

What I see is that the `$appPath` is not take in consideration. In my example I have set the `$appPath` to an empty string and I get the document printed.

Thanks a lot for the quick answer.
